src/python/pants/backend/python/lint/bandit/rules_integration_test.py

```diff
@@ -172,6 +173,7 @@ def test_skip(self) -> None:
         result = self.run_bandit([target], skip=True)
         assert not result
 
+    @skip("flaky depending on Python version: https://github.com/pantsbuild/pants/issues/10545")
```

Comment:
@Eric-Arellano has suggested that rather than skipping this, it would be good to try to pin the interpreters that we use with bandit: ie:
```
diff --git a/src/python/pants/backend/python/lint/bandit/rules_integration_test.py b/src/python/pants/backend/python/lint/bandit/rules_integration_test.py
index 8def49ada..219f046b1 100644
--- a/src/python/pants/backend/python/lint/bandit/rules_integration_test.py
+++ b/src/python/pants/backend/python/lint/bandit/rules_integration_test.py
@@ -174,7 +174,10 @@ class BanditIntegrationTest(ExternalToolTestBase):
     def test_3rdparty_plugin(self) -> None:
         target = self.make_target_with_origin(
-            [FileContent("bad.py", b"aws_key = 'JalrXUtnFEMI/K7MDENG/bPxRfiCYzEXAMPLEKEY'\n")]
+            [FileContent("bad.py", b"aws_key = 'JalrXUtnFEMI/K7MDENG/bPxRfiCYzEXAMPLEKEY'\n")],
+            # NB: `bandit-aws` does not currently work with Python 3.8. See
+            #  https://github.com/pantsbuild/pants/issues/10545.
+            interpreter_constraints="CPython>=3.6,<3.8",
         )
         result = self.run_bandit(
             [target], additional_args=["--bandit-extra-requirements=bandit-aws"]
```

Reply:
I'll put the fix in the separate PRs I opened to skip the tests (and rename the PRs).